gui/webdisplay/src/RWebDisplayArgs.cxx

```diff
@@ -143,15 +143,17 @@ bool RWebDisplayArgs::SetPosAsStr(const std::string &str)
 ///
 /// Recognized values:
 ///
-///      chrome - use Google Chrome web browser, supports headless mode from v60, default
-///     firefox - use Mozilla Firefox browser, supports headless mode from v57
-///      native - (or empty string) either chrome or firefox, only these browsers support batch (headless) mode
-///     browser - default system web-browser, no batch mode
-///      safari - Safari browser on Mac
+///      chrome - use Google Chrome web browser
+///     firefox - use Mozilla Firefox browser
+///        edge - use Edge web browser (Windows only)
+///      native - either chrome or firefox, only these browsers support batch (headless) mode
```

Comment:
what happens in N years when edge changes name? Can we be more future proof and write "(curently edge)"?